py-polars/docs/source/reference/expressions/boolean.rst

```diff
@@ -14,6 +14,7 @@ Boolean
     Expr.is_first
     Expr.is_in
     Expr.is_infinite
+    Expr.is_last
```

Comment:
I don't know if the `.rst` file is automatically generated. If so, what commands do I need to execute?

Reply:
It's not! These need to be added manually.